Makefile

```diff
@@ -210,7 +210,19 @@ chlog-update: $(CHLOGGEN)
 # files that have the "area" dropdown field
 .PHONY: generate-gh-issue-templates
 generate-gh-issue-templates:
-	$(TOOLS_DIR)/scripts/update-issue-template-areas.sh
+	mkdir -p $(TOOLS_DIR)/bin
+	docker run --rm \
+	-u $(id -u ${USER}):$(id -g ${USER}) \
+	--mount 'type=bind,source=$(PWD)/internal/tools/scripts,target=/home/weaver/templates,readonly' \
+	--mount 'type=bind,source=$(PWD)/model,target=/home/weaver/source,readonly' \
+	--mount 'type=bind,source=$(TOOLS_DIR)/bin,target=/home/weaver/target' \
+	$(WEAVER_CONTAINER) registry generate \
+		--registry=/home/weaver/source \
+		--templates=/home/weaver/templates \
+		--config=/home/weaver/templates/registry/areas-weaver.yaml \
```

Comment:
Fun use of weaver !

Reply:
Indeed!

Reply:
It's really cool to be able to get a list of actual namespaces instead of relying on the folder structure!